build.fsx

```diff
@@ -876,7 +878,8 @@ Target.create "DotNetCreateNuGetPackage" (fun _ ->
                     { c.MSBuildParams with
                         Properties =
                             [ ("Version", nugetVersion)
-                              ("PackageReleaseNotes", release.Notes |> String.toLines) ] } }
+                              ("PackageReleaseNotes", release.Notes |> String.toLines) ] 
+                        DisableInternalBinLog = true } }
```

Comment:
Why do we need this?
Maybe explanatory comment will be relevant here?

Reply:
This was due to issues generated of updated MSBuild.StructuredLogger:
- If it's not updated, the run will fail with "Unsupported log file format. Latest supported version is"
- If you user the updated version without it, it says "DotNet.publish fails attempting to load Microsoft.Build.Framework"

But there is even more recent version now, so I'm not sure if it's needed anymore or is the issue fixed.

Reply:
I tested, yes it was needed. Comment added.